openforcefield/content/consortium/minutes/2018-10-11-minutes.md

```diff
@@ -0,0 +1,41 @@
+---
+date: "2018-10-11T00:00:00+00:00"
+title: "October 11, 2018 Governing Board Meetings and notes"
+tags: ["general"]
+categories: ["minutes"]
+draft: false
+description: "October 11, 2018 Governing Board Meetings and notes"
+weight: 10
+author: "Lee-Ping Wang"
+---
+
+## Agenda: 
+- October meeting draft agenda and start time
+- January meeting location
+- Maat hiring
+- Slack
+- Funding processing update/onboarding update
+- Involvement with science meetings (subprojects) and project planning 
+- Discuss blog posts? Yes, just get sub-groups to do brief updates after meetings; better for keeping people up to date.
+
+## Minutes:
+Attending: All PIs except Shirts, and both industry members, Thomas Fox and Katharina Meier
```

Comment:
I made comments/suggested changes above; they would need to be reflected here too.